L/LAPACK/common.jl

```diff
@@ -2,7 +2,7 @@ using BinaryBuilder, Pkg
 
 # LAPACK mirrors the OpenBLAS build, whereas LAPACK32 mirrors the OpenBLAS32 build.
 
-version = v"3.12.0"
+version = v"3.12.1"
 
 # Collection of sources required to build lapack
 sources = [
```

Comment:
I think you need to update the git tag here to point to the release's git commit now.